charts/rstudio-pm/values.yaml

```diff
@@ -191,3 +191,6 @@ config:
   Launcher:
     ServerUser: root
     AdminGroup: root
+
+# -- sidecar container list
+sidecarContainers: []
```

Comment:
Please use `extraContainers` instead, and update existing references. I've also added suggested documentation, from looking at a few other sources:

```suggestion
# -- Additional containers to be added to the pod.
extraContainers: []
```